README.md

```diff
@@ -9,6 +9,8 @@ This is the Svelte compiler, which is primarily intended for authors of tooling
 * [svelte-cli](https://github.com/sveltejs/svelte-cli) – Command line interface for compiling components
 * [rollup-plugin-svelte](https://github.com/rollup/rollup-plugin-svelte) – Rollup plugin
 * [sveltify](https://github.com/tehshrike/sveltify) - Browserify transform
+* [gulp-svelte](https://github.com/shinnn/gulp-svelte) - gulp plugin
```

Comment:
@Rich-Harris Do you have any preference of the link order? I just added them to the bottom, following https://github.com/sveltejs/svelte/pull/83.